### ISSUE TITLE
Pain on the medial aspect of the foot

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11180]
**Preferred term label:**
Pain on the medial aspect of the foot

**Synonyms**

**Definition (free text, please give PubMed ID)**
Tenderness to palpation of medial aspect of the foot

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0025238

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Calcaneus fractures
Hallux valgus, adult
Tarsal tunnel syndrome
Plantar fasciitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001402
- label: Pain on the medial aspect of the foot
- parents: [Foot pain (HP:0025238)](https://hpo.jax.org/app/browse/term/HP:0025238)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the side of the foot nearest to the big toe (hallux).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11180
- synonyms:
